### ISSUE TITLE
Increase MEMPOOL_SIZE

### ISSUE THREAD

**Issue by dconnolly** [ZcashFoundation/zebra#2675]
## Motivation

Right now our mempool is very small, only max two (2) transactions at any time. This means that when we evict a tx from the mempool because we are out of space, we track it in the rejected set as State::Victim. For the smaller mempool we have now, we may be evicting 'innocent victims' frequently, that we may actually want to verify and gossip to peers. This is less of an issue with a larger mempool.

## Related Work

This is blocked by https://github.com/ZcashFoundation/zebra/issues/2637 and https://github.com/ZcashFoundation/zebra/issues/2592 / https://github.com/ZcashFoundation/zebra/issues/2603

**Comment by dconnolly** [comment]
This may be resolve by completing https://github.com/ZcashFoundation/zebra/issues/2744

**Comment by teor2345** [comment]
We moved the eviction tickets to the next sprint, so I'm going to move this one too.

**Comment by teor2345** [comment]
Actually, I think this is now effectively a duplicate of #2744, so I'm just going to close it.
